engine/src/test/java/com/redhat/cloud/notifications/recipients/rbac/RbacRecipientUsersProviderTest.java

```diff
@@ -122,6 +123,16 @@ public void getAllUsersFromDefaultGroup() {
         }
     }
 
+
+    @Test
+    public void getUsersFromNonExistentGroupResultsInNoUsers() {
```

Comment:
I personally prefer the "pattern" should<<what I expect>>When<<What I do>> as test name. If the test fails and people read the failed test in the console or in the IDE, they immediately know what is going on in the test, what's expected and so on.

Something like this as test name maybe "shouldReturnNoUsersWhenGroupNotFound". 

This is again up to you to change it :-)

Reply:
It's just my personal preference. There are lots of different patterns for naming unit tests: https://dzone.com/articles/7-popular-unit-test-naming

Maybe that's something we should talk about with the other devs and then decide if we prefer one strategie? 
With one naming strategy the codebase would look a bit "cleaner"